### PR TITLE
EES-1008: set all log levels to information

### PR DESCRIPTION
**Pull request by jamescutts**

### CODE REVIEW

File: src/GovUk.Education.ExploreEducationStatistics.Admin/appsettings.json [hunk 1]
```diff
@@ -1,7 +1,12 @@
 {
   "Logging": {
     "LogLevel": {
-      "Default": "Debug"
+      "Default": "Information"
+    },
+    "ApplicationInsights": {
+      "LogLevel": {
+        "Default": "Information"
```
Comment (benoutram) [inline src/GovUk.Education.ExploreEducationStatistics.Admin/appsettings.json:8]: This conflicts with the following line already in Startup.cs
`builder.AddFilter<ApplicationInsightsLoggerProvider>("", HostingEnvironment.IsDevelopment() ? LogLevel.Debug : LogLevel.Warning);`

File: src/GovUk.Education.ExploreEducationStatistics.Notifier/host.json [hunk 2]
```diff
@@ -1,3 +1,8 @@
 {
-    "version": "2.0"
+    "version": "2.0",
+    "logging": {
+        "logLevel": {
```
Comment (benoutram) [inline src/GovUk.Education.ExploreEducationStatistics.Notifier/host.json:4]: Mistake in `logLevel` here?

File: src/GovUk.Education.ExploreEducationStatistics.Data.Api/appsettings.json [hunk 3]
```diff
@@ -2,6 +2,11 @@
   "Logging": {
     "LogLevel": {
       "Default": "Information"
+    },
+    "ApplicationInsights": {
```
Comment (benoutram) [inline src/GovUk.Education.ExploreEducationStatistics.Data.Api/appsettings.json:6]: Data API doesn't have this line which Admin has. Does it need it?
`builder.AddApplicationInsights(Configuration.GetSection("AppInsights").GetValue<string>("InstrumentationKey"));`

If not, what's the difference?

File: src/GovUk.Education.ExploreEducationStatistics.Notifier/host.json (hunk 2 above)
Comment (jamescutts) [reply src/GovUk.Education.ExploreEducationStatistics.Notifier/host.json:4]: no, functions host.js is slightly different in its formatting for some reason

File: src/GovUk.Education.ExploreEducationStatistics.Data.Api/appsettings.json (hunk 3 above)
Comment (jamescutts) [reply src/GovUk.Education.ExploreEducationStatistics.Data.Api/appsettings.json:6]: This is covered under a separate piece of work reviewing how app insights should be configured

File: src/GovUk.Education.ExploreEducationStatistics.Admin/appsettings.json (hunk 1 above)
Comment (jamescutts) [reply src/GovUk.Education.ExploreEducationStatistics.Admin/appsettings.json:8]: Resolved

File: src/GovUk.Education.ExploreEducationStatistics.Admin/Startup.cs [hunk 4]
```diff
@@ -72,6 +72,14 @@ public Startup(IConfiguration configuration, IHostingEnvironment hostingEnvironm
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+            services.AddLogging(builder =>
+            {
+                builder.AddApplicationInsights(Configuration.GetSection("AppInsights").GetValue<string>("InstrumentationKey"));
+                builder.AddFilter<ApplicationInsightsLoggerProvider>("", HostingEnvironment.IsDevelopment() ? LogLevel.Debug : LogLevel.Information);
+                builder.AddFilter<ApplicationInsightsLoggerProvider>("Microsoft", LogLevel.Information);
```
Comment (benoutram) [inline src/GovUk.Education.ExploreEducationStatistics.Admin/Startup.cs:79]: Logging filtering can be done programatically or in appsettings.json (we also have appsettings.Development.json as an override for the Development profile) so I'm not sure why we are setting log levels in code and in the json. Here we have category `"Microsoft"` set in the code but not in the json so somebody expecting to find the log levels from just the json will miss it unless they know to check here too and evaluate the overall combined settings. It's confusing so can you remove it please?

Comment (jamescutts) [reply src/GovUk.Education.ExploreEducationStatistics.Admin/Startup.cs:79]: This is being looked at as part of wider working around tidying up our logging, this is simple setting things more consistiently

**Pull request merged**